src/GovUk.Education.ExploreEducationStatistics.Content.Services/ReleaseFileService.cs

```diff
@@ -121,7 +125,10 @@ public class ReleaseFileService : IReleaseFileService
                     continue;
                 }
 
-                var entry = archive.CreateEntry(releaseFile.File.ZipFileEntryName());
+                var entry = archive
+                    .CreateEntry(releaseFile.File.ZipFileEntryName())
+                    .SetUnixPermissions("664");
```

Comment:
What effect does this have on non-Unix OS's?

Reply:
None as far as I've seen! File permissions appear to be fine for Windows